testsuite/single_node_performance.py

```diff
@@ -14,24 +14,24 @@
 # numbers are based on the machine spec used by github action
 # Local machine numbers will be higher.
 EXPECTED_TPS = {
-    ("no-op", False): (18200.0, True),
-    ("coin-transfer", False): (11800.0, True),
-    ("coin-transfer", True): (20000.0, True),
-    ("account-generation", False): (9900.0, True),
-    ("account-generation", True): (16300.0, True),
-    ("create-new-account-resource", False): (11700.0, True),
+    ("no-op", False): (19000.0, True),
+    ("coin-transfer", False): (12500.0, True),
+    ("coin-transfer", True): (22000.0, True),
+    ("account-generation", False): (11000.0, True),
+    ("account-generation", True): (17500.0, True),
+    ("create-new-account-resource", False): (12500.0, True),
     ("modify-global-resource", False): (4000.0, True),
-    ("modify-ten-global-resources", False): (10500.0, True),
-    ("large-module-working-set-no-op", False): (2550.0, True),
+    ("modify-ten-global-resources", False): (11000.0, True),
+    ("large-module-working-set-no-op", False): (3000.0, True),
     ("publish-package", False): (130.0, True),
-    ("batch100-transfer", False): (300, True),
-    ("batch100-transfer", True): (500, True),
+    ("batch100-transfer", False): (330, True),
+    ("batch100-transfer", True): (550, True),
     # ("token-v1ft-mint-and-store", False): 1000.0,
     ("token-v1ft-mint-and-transfer", False): (1700.0, True),
     ("token-v1nft-mint-and-transfer-sequential", False): (1100.0, True),
-    ("token-v1ft-mint-and-transfer20-collections", False): (6400.0, False),
-    ("token-v1nft-mint-and-transfer-sequential20-collections", False): (4400.0, False),
-    ("token-v1nft-mint-and-transfer-parallel", False): (1000.0, False),
+    ("token-v1ft-mint-and-transfer20-collections", False): (7000.0, False),
+    ("token-v1nft-mint-and-transfer-sequential20-collections", False): (5200.0, False),
+    ("token-v1nft-mint-and-transfer-parallel", False): (1400.0, False),
```

Comment:
Does the other two tests change in performance?

Can you paste comparisons between your PR and another PR to show the actural improvements?

Reply:
Well, took another look, and it seems the tps doesn't really reflect the improvement of this PR, added more details in the PR description.